_opensearch/rest-api/create-alias.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Create alias
+parent: REST API reference
+grand_parent: OpenSearch
+nav_order: 8
+---
+
+# Create alias
+
+An alias is a virtual pointer that you can use to reference one or more indices. For example, if you have continuous streams of incoming data but only want to search through the past seven days at any time, you can use an alias to point to only indices created within the past seven days.
```

Comment:
The big thing here is that aliases give your clients a consistent path, so they can always just search "logs-this-week/_search" instead of having to figure out the day and provide the correct index names.